Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -13,6 +13,7 @@
   },
   "AllowedHosts": "*",
   "BearerTokenTimeoutMinutes" : "30",
+  "SecurityMetadataCacheTimeoutMinutes": "0",
   "DefaultPageSizeLimit" : 500,
   "ApiSettings": {
```

Comment:
Should MinimalTemplateSuffix and PopulatedTemplateSuffix defaults migrate up to here from the development file?  @vimayya Related, should we have packaged defaults for the Minimal/Populated TemplateScript?

Reply:
@blmeyers , @raghuramakrishna  yes. I have noted that in the comments here ODS-4622, I would prefer to have default local connection strings as well.